### PR TITLE
SYN-6814: Add note about Synapse not a data lake

### PR DESCRIPTION
**Pull request by MichaelSquires**
Updated the introduction page with a note about Synapse not being a data lake. 

Preview:
image

### CODE REVIEW

**Comment by therealsilence** [comment]
looks good to me :-)

**Review by therealsilence — APPROVED** [review]

File: docs/synapse/intro.rst [hunk 1]
```diff
@@ -17,6 +17,13 @@ structured and extensible :ref:`userguide_datamodel` and the powerful and intuit
 query language, Synapse gives analysts unparalleled power and flexibility to ask and answer any question,
 even over large and complex data sets.
 
+.. note::
+
+    A :ref:`gloss-cortex` may easily grow to billions of nodes, but is not designed to consume and create billions of
+    nodes **per day**. In other words, Synapse is not meant to replace your big-data/data-lake storage; Synapse is
+    designed to connect to your data sources on demand in order to ingest data **relevant for your analysis** into the
```
Comment (invisig0th) [inline docs/synapse/intro.rst:24]:
```suggestion
    designed to connect to your data sources on demand in order to ingest data relevant for your analysis into the
```
No need to be quite so forceful ;)

File: docs/synapse/intro.rst [hunk 2]
```diff
@@ -17,6 +17,13 @@ structured and extensible :ref:`userguide_datamodel` and the powerful and intuit
 query language, Synapse gives analysts unparalleled power and flexibility to ask and answer any question,
 even over large and complex data sets.
 
+.. note::
+
+    A :ref:`gloss-cortex` may easily grow to billions of nodes, but is not designed to consume and create billions of
+    nodes **per day**. In other words, Synapse is not meant to replace your big-data/data-lake storage; Synapse is
```
Comment (invisig0th) [inline docs/synapse/intro.rst:23]:
```suggestion
    nodes per day. In other words, Synapse is not meant to replace your big-data/data-lake storage; Synapse is
```

**Review by invisig0th — CHANGES_REQUESTED** [review]
nits ;)

**Review by invisig0th — APPROVED** [review]

**Pull request merged**